src/pages/form-validation/verify-on-submit/index.tsx

```diff
@@ -0,0 +1,243 @@
+    const [serialNumber, setSerialNumber] = useState('');
+    const [errorMsg, setErrorMsg] = useState('');
+    const [deviceAdded, setDeviceAdded] = useState(false);
+    const [showDeviceAddedScreen, setShowDeviceAddedScreen] = useState(false);
+    const [showAddDeviceScreen, setShowAddDeviceScreen] = useState(true);
+    const [loading, setLoading] = useState(false);
+    const inputId = 'serialNumber';
+    const slideAnimationDurationMs = 250;
+
+    useEffect(() => {
+        const input = document.getElementById(inputId);
+        if (input) {
+            input.focus();
+        }
+    }, []);
+
+    const onSerialNumberChange: OnChangeHandler = useCallback((event) => {
+        setSerialNumber(event.target.value);
+    }, []);
+
+    const onSubmit = (serial: string): void => {
+        setErrorMsg('');
+        setLoading(true);
+        setTimeout(() => {
+            setLoading(false);
+            if (serial !== '123') {
+                setErrorMsg('Device not found.');
+                const input = document.getElementById(inputId);
+                if (input) {
+                    input.focus();
+                }
+            } else {
+                setDeviceAdded(true);
+                setShowAddDeviceScreen(false);
+                setTimeout(() => {
+                    setShowDeviceAddedScreen(true);
+                }, slideAnimationDurationMs);
+            }
+        }, 2000);
+    };
+
+    return (
+        <>
+            <AppBar data-cy="pxb-toolbar" position={'sticky'} classes={{ root: classes.appbarRoot }}>
+                <Toolbar classes={{ gutters: classes.toolbarGutters }}>
+                    <Hidden mdUp>
+                        <IconButton
+                            data-cy="toolbar-menu"
+                            color={'inherit'}
+                            onClick={(): void => {
+                                dispatch({ type: TOGGLE_DRAWER, payload: true });
+                            }}
+                            edge={'start'}
+                            style={{ marginRight: 20 }}
+                        >
+                            <Menu />
+                        </IconButton>
+                    </Hidden>
+                    <Typography variant={'h6'} color={'inherit'}>
+                        Verify on Submit
+                    </Typography>
+                </Toolbar>
+            </AppBar>
+
+            <div className={classes.containerWrapper}>
+                <div className={classes.container}>
+                    <Slide
+                        direction="right"
+                        in={showAddDeviceScreen}
+                        mountOnEnter
+                        unmountOnExit
+                        timeout={deviceAdded ? slideAnimationDurationMs : 0}
+                    >
+                        <div>
+                            <Typography variant={'h6'} style={{ marginBottom: theme.spacing(2) }}>
+                                Find Device
```

Comment:
Can we change button text `Find Device`  => `Search Device` ?

Reply:
Ok this is updated.